### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,106 @@
...
+    "version": "dc231ba",
+    "assets_version": "dc231ba",
+    "owner": "bill"
+  }
+}
+```
+
+Response:
+```
+Status: 201 Created
```
Comment (bobbytables) [inline README.md]: This should probably be a 202.

Comment (macat) [inline README.md]: Why 202?

Comment (macat) [inline README.md]: I think it should be 201 if it's a new object and 200 if it's an update.

Comment (bobbytables) [inline README.md]: because a 202 is a deferred create. The effect of this request is not immediately noticeable right?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,106 @@
 # broadway
 Broadway 
+
+Broadway helps developers deploy their projects with defining their deployment
+workflows in Broadway playbooks. It runs as a service interacting with
+Kubernetes and allowing users to interact with it through various interfaces
+(Slack, CLI, Web).
```
Comment (bobbytables) [inline README.md]: :+1: :fire:

Comment (cored) [inline README.md]: Broadway allows that but through the API, not specified to any client

File: README.md (hunk 1 above)
Comment (macat) [inline README.md]: The resource (instance) will be created. The user need fire off a deploy call to deploy it.

File: README.md (hunk 2 above)
Comment (macat) [inline README.md]: What?

File: README.md (hunk 1 above)
Comment (bobbytables) [inline README.md]: Ah I see.

File: README.md (hunk 2 above)
Comment (bobbytables) [inline README.md]: Saying I like it!

**Comment by bobbytables** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (macat) [inline README.md]: Rafael had a comment here, that's why I was asking.

**Comment by poopoothegorilla** [comment]
+1

**Pull request merged**